serverless/install/install-serverless-operator.adoc

```diff
@@ -7,35 +7,39 @@ include::modules/common-attributes.adoc[]
+
+Read the following information about supported configurations and prerequisites before you install {ServerlessProductName}.
+
+* {ServerlessProductName} is supported for installation in a restricted network environment.
 
-[NOTE]
+* {ServerlessProductName} currently cannot be used in a multi-tenant configuration on a single cluster.
+
+[IMPORTANT]
 ====
-{ServerlessProductName} currently cannot be used in a multi-tenant configuration on a single cluster.
+Before upgrading to the latest Serverless release, you must remove the community Knative Eventing Operator if you have previously installed it. Having the Knative Eventing Operator installed prevents you from being able to install the latest version of Knative Eventing using the {ServerlessOperatorName}.
 ====
 
-include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+1]
+include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+2]
+
+[id="install-serverless-operator-scaling-with-machinesets"]
+=== Scaling your cluster using machine sets
 
-If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200MB - 2GB of memory for each replica of the Knative Serving control plane.
+You can use the {product-title} `MachineSet` API to manually scale your cluster up to the desired size. The minimum requirements usually mean that you must scale up one of the default machine sets by two additional machines. See xref:../../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[Manually scaling a machine set].
 
-HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../../serverless/admin_guide/serverless-ha.adoc#serverless-ha[Configuring high availability replicas on {ServerlessProductName}].
+include::modules/serverless-install-web-console.adoc[leveloffset=+1]
 
-[id="installing-openshift-serverless-scaling-with-machinesets"]
-== Scaling your cluster using machine sets
+[id="additional-resources_install-serverless-operator"]
+== Additional resources
 
-You can use the {product-title} `MachineSet` API to manually scale your cluster up to the desired size.
-The minimum requirements usually mean that you must scale up one of the default machine sets by two additional machines.
-See xref:../../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[Manually scaling a machine set].
+* For more information about using {ServerlessProductName} on a restricted network environment, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
 
-include::modules/serverless-install-web-console.adoc[leveloffset=+1]
+* HA is enabled for some Knative components by default. You can disable HA by following the documentation on xref:../../serverless/admin_guide/serverless-ha.adoc#serverless-ha[Configuring high availability replicas on {ServerlessProductName}].
```

Comment:
Here.. I see the linked doc does not explain how to disable HA, but how to configure number of replicas.. so I'm wondering if that should be re-phased.. what about ...
```suggestion
* HA is enabled for some Knative components by default. You can configure HA by following the documentation on xref:../../serverless/admin_guide/serverless-ha.adoc#serverless-ha[Configuring high availability replicas on {ServerlessProductName}].
```

Reply:
Good call. We should also do a refresh of the HA docs at some point but I'll leave it alone for now since it's not in scope yet.